md/workspaces-and-repositories.md

```diff
@@ -95,21 +95,23 @@ If the list does not contain the name of the repository you want to use, check t
 
 #### Git integration in Studio 
 
-Bonita studio Git integration is based on th EGit eclipse plugin.
+Bonita Studio Git integration is based on the EGit Eclipse plugin.
 
-Git command available in Studio interface:
+Git commands available in the studio interface:
 
  * **Share with Git**  
 This action connects the current repository to Git and shares it on a remote.
 To configure the remote, see the following [Egit userguide](http://wiki.eclipse.org/EGit/User_Guide#Working_with_remote_Repositories) or the [Share on GitHub howto](share-a-repository-on-github.md).
 
  * **Clone**  
-Create a new Studio repository from an existing Git repository (that must contain a proper Bonita project). If the remote repository version is lower than the Studio, a migration will be applied on the cloned repository. Be careful before pushing a migrated repository back to the remote: all contributors will have to use the proper Studio version.
-If you used Bonita on Git before Bonita 7.7.0, you might want to clone it directly from the Studio. For that, your project on github must be Bonita compliant:
+Create a new Studio repository from an existing Git repository (that must contain a proper Bonita project). If the remote repository version is lower than the studio, a migration will be applied on the cloned repository. Be careful before pushing a migrated repository back to the remote: all contributors will have to use the proper studio version.
+If you used Bonita on Git before Bonita 7.7.0, you might want to clone it directly from the studio. To do so, your project on GitHub must be "Bonita compliant":
 
 	 * The Git repository must correspond to a Bonita repository (and not the Bonita Workspace)
 	 * The .project file must be present 
-	 * It is highly recommended to use the .gitignore file generated by Bonita when you share a Bonita repository from the Studio.
+	 * It is highly recommended to use the .gitignore file generated by Bonita when you share a Bonita repository from the studio.
+
+Despite those conditions, we cannot guarantee that such a cloning works properly. If you encounter issues on the cloned repository, you may want to export the repository from Git and import it in the Studio, to share it back to a remote repository.
```

Comment:
What does it mean? export from git and import in the Studio? You cannot create a .bos from github, and you cannot import a repository